src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,89 @@
 import { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
-  // Increase the count after successful form submission
-  // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+interface NewMovieProps {
+  onAdd: (movie: Movie) => void;
+}
+
+export const NewMovie: React.FC<NewMovieProps> = ({ onAdd }) => {
+  const [count, setCount] = useState(0);
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [imgUrl, setImgUrl] = useState('');
+  const [imdbUrl, setImdbUrl] = useState('');
+  const [imdbId, setImdbId] = useState('');
+
+  const disabledSubmit = !title || !imgUrl || !imdbUrl || !imdbId;
```

Comment:
```suggestion
  const isSubmitDisabled = !title || !imgUrl || !imdbUrl || !imdbId;
```
It would be great to name boolean variables according to these rules, so it's clear that they store a boolean
![image](https://github.com/mate-academy/react_movies-list-add-form/assets/96409563/b350b23c-7ac1-4b4a-befe-f1b8fd3cb9a2)